examples/spire-sidecar/sidecar/run-sidecar.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Do we really use this script anymore? I think we don't need it as long as `run-sidecar-bash.sh` is there.

Reply:
not needed, can be removed